terraform/modules/apps/router/main.tf

```diff
@@ -0,0 +1,77 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 2.69"
+    }
+  }
+}
+
+data "aws_secretsmanager_secret" "sentry_dsn" {
+  name = "router_app-SENTRY_DSN"
+}
+
+module "app" {
+  source                           = "../../app"
+  cpu                              = 512
+  memory                           = 1024
+  vpc_id                           = var.vpc_id
+  cluster_id                       = var.cluster_id
+  service_name                     = var.service_name
+  subnets                          = var.private_subnets
+  mesh_name                        = var.mesh_name
+  service_discovery_namespace_id   = var.service_discovery_namespace_id
+  service_discovery_namespace_name = var.service_discovery_namespace_name
+  task_role_arn                    = var.task_role_arn
+  execution_role_arn               = var.execution_role_arn
+  extra_security_groups            = [var.govuk_management_access_sg_id]
+  container_definitions = [
+    {
+      "name" : "router",
+      "image" : "govuk/router:deployed-to-production",
+      "essential" : true,
+      "environment" : [
+        { "name" : "GOVUK_APP_NAME", "value" : "router" },
+        { "name" : "GOVUK_APP_ROOT", "value" : "/var/apps/router" },
+        { "name" : "ROUTER_APIADDR", "value" : ":8081" },
+        { "name" : "ROUTER_BACKEND_HEADER_TIMEOUT", "value" : "20s" },
+        { "name" : "ROUTER_PUBADDR", "value" : ":8080" },
+        { "name" : "ROUTER_MONGO_DB", "value" : "router" },
+        { "name" : "ROUTER_MONGO_URL", "value" : "mongodb://${var.router_mongodb_host}" },
+        { "name" : "SENTRY_ENVIRONMENT", "value" : "test" }
```

Comment:
I just realised this is going to need to vary by environment, but no need to block merging - easily done as a follow-up.